content/v2.0/write-data/csv.md

```diff
@@ -0,0 +1,445 @@
+or data is incorrectly formatted, when processing the row, the `influx write` command
+returns an error and cancels the write request.
+To skip rows with errors, use the `--skipRowOnError` flag.
+
+```sh
+influx write -b example-bucket \
+  -f path/to/example.csv \
+  --skipRowOnError
+```
+
+{{% warn %}}
+Skipped rows are ignored and are not written to InfluxDB.
+{{% /warn %}}
+
+## Advanced examples
+
+- [Define constants](#define-constants)
+- [Annotation shorthand](#annotation-shorthand)
+- [Use alternate numeric formats](#use-alternate-numeric-formats)
+- [Use alternate boolean format](#use-alternate-boolean-format)
+- [Use different timestamp formats](#use-different-timestamp-formats)
+
+---
+
+### Define constants
+Use the Extended annotated CSV [`#constant` annotation](/v2.0/reference/syntax/annotated-csv/extended/#constant)
+to add a column and value to each row in the CSV data.
+
+{{< flex >}}
+{{% flex-content %}}
+##### CSV with constants
+```
+#constant measurement,example
+#constant tag,source,csv
+#datatype long,dateTime:RFC3339
+count,time
+1,2020-01-01T00:00:00Z
+4,2020-01-02T00:00:00Z
+9,2020-01-03T00:00:00Z
+18,2020-01-04T00:00:00Z
+```
+{{% /flex-content %}}
+{{% flex-content %}}
+##### Resulting line protocol
+```
+example,source=csv count=1 1577836800000000000
+example,source=csv count=4 1577923200000000000
+example,source=csv count=9 1578009600000000000
+example,source=csv count=18 1578096000000000000
+```
+{{% /flex-content %}}
+{{< /flex >}}
+
+---
+
+### Annotation shorthand
+Extended annotated CSV supports [annotation shorthand](/v2.0/reference/syntax/annotated-csv/extended/#annotation-shorthand),
+which lets you define the **column label**, **datatype**, and **default value** in the column header.
+
+{{< flex >}}
+{{% flex-content %}}
+##### CSV with annotation shorthand
+```
+m|measurement,count|long|0,time|dateTime:RFC3339
+example,1,2020-01-01T00:00:00Z
+example,4,2020-01-02T00:00:00Z
+example,,2020-01-03T00:00:00Z
+example,18,2020-01-04T00:00:00Z
+```
+{{% /flex-content %}}
+{{% flex-content %}}
+##### Resulting line protocol
+```
+example count=1 1577836800000000000
+example count=4 1577923200000000000
+example count=0 1578009600000000000
+example count=18 1578096000000000000
+```
+{{% /flex-content %}}
+{{< /flex >}}
+
+#### Replace column header with annotation shorthand
+It's possible to replace the column header row in a CSV file with annotation
+shorthand without modifying the CSV file.
+This lets you define column data types and default values while writing to InfluxDB.
+
+To replace an existing column header row with annotation shorthand:
+
+1. Use the `--skipHeader` flag to ignore the existing column header row.
+2. Use the `--header` flag to inject a new column header row that uses annotation shorthand.
+
+<!-- -->
+```sh
+influx write -b example-bucket \
+  -f example.csv \
+  --skipHeader=1
+  --header="m|measurement,count|long|0,time|dateTime:RFC3339"
```

Comment:
note: you can also save all injected headers to (an UTF-8 encoded) file `headers` and then import two files. The `--skipHeader` option applies only to the last input (file or stdin)

```
influx write -b example-bucket \
  -f headers.csv \
  -f example.csv \
  --skipHeader=1
```

Reply:
Awesome. I added this to the "Inject headers" section.